hgvs/dataproviders/uta.py

```diff
@@ -623,8 +623,9 @@ def _get_cursor(self, n_retries=1):
                 _logger.warning(
                     "Lost connection to {url}; attempting reconnect".format(url=self.url))
                 if self.pooling:
-                    self._pool.closeall()
-                self._connect()
+                    self._pool.putconn(conn)
+                else:
+                    self._connect()
                 _logger.warning("Reconnected to {url}".format(url=self.url))
```

Comment:
I would suggest indenting this so it only outputs the warning in the non-pooled case after the `_connect()`. In the pooled case we aren't actually reconnected here, we're just prepared for a reconnect on the next use.